integration/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "graphback-integration-tests",
+  "version": "0.10.0-rc2",
+  "private": true,
+  "description": "Integration tests",
+  "main": "index.js",
+  "scripts": {
+    "develop": "echo disabled",
+    "build": "echo disabled",
+    "test": "ava",
+    "lint": "echo disabled",
+    "start": "echo disabled",
+    "pretest": "npm run build"
+  },
+  "dependencies": {
+    "apollo-boost": "^0.4.4",
+    "graphback": "0.10.0-rc2",
+    "graphql": "^14.0.0",
+    "graphql-testx": "0.3.0-dev381.bede8a3",
```

```suggestion
    "graphql-testx": "0.3.0-dev384.3a11247",
```
Just released the a new testx version with the new `Database` interface, we can easily integrate it, it's just a matter of rename a couple of objects and imports, but no cahges to the logic